src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -208,5 +209,55 @@ public string Value
         ///   Returns <see cref="JsonValueKind.String"/>
         /// </summary>
         public override JsonValueKind ValueKind { get => JsonValueKind.String; }
+
+        /// <summary>
+        ///   Converts the text value of this instance, which should encode binary data as base-64 digits, to an equivalent 8-bit unsigned <see cref="byte"/> array.
+        ///   The return value indicates wether the conversion succeeded.
+        /// </summary>
+        /// <param name="value">
+        ///   When this method returns, contains the <see cref="byte"/> array equivalent of the text contained in this instance,
+        ///   if the conversion succeeded.
+        /// </param>
+        /// <returns>
+        ///   <see langword="true"/> if text was converted successfully; othwerwise returns <see langword="false"/>.
+        /// </returns>
+        internal bool TryGetBytesFromBase64(out byte[] value)
+        {
+            if (_value.Length < 1)
+            {
+                value = default;
+                return false;
+            }
+
+            // we decode string -> byte, so the resulting length will
+            // be /4 * 3 - padding. To be on the safe side, keep padding in slice later
+            int bufferSize = _value.Length / 4 * 3;
+
+            byte[] arrayToReturnToPool = null;
+            try
+            {
+                Span<byte> buffer = bufferSize <= 256
```

Comment:
We should add tests to cover the different branches here (add cases where the bufferSize <= 256 and > 256). If you run test coverage, you should be able to see the missing branches.

Run the following from the `E:\GitHub\Fork\corefx\src\System.Text.Json\tests` directory
```text
dontet msbuild /t:BuildAndTest /p:Coverage=true
```

https://github.com/dotnet/corefx/blob/4b6562e0da17ba1d92d72bc2406af9eea3e9337a/Documentation/building/code-coverage.md#local-code-coverage-runs

Reply:
I added a test case to resolve this in my recent commit. Great catch!